qutebrowser/config/configdata.yml

```
@@ -1396,12 +1396,16 @@ editor.command:
     * `{line0}`: Same as `{line}`, but starting from index 0.
     * `{column0}`: Same as `{column}`, but starting from index 0.
 
-
 editor.encoding:
   type: Encoding
   default: utf-8
   desc: Encoding to use for the editor.
 
+editor.remove_file:
+  default: true
+  type: Bool
+  desc: Whether to delete the temporary file upon closing the editor.
```

Could you please switch this to imperative mood for consistency with other Boolean configuration options?
See https://github.com/qutebrowser/qutebrowser/commit/494aceec451e307b0227b0fdd0f86d419dbc11d1 for examples.

```suggestion
  desc: Delete the temporary file upon closing the editor.
```

---

Done!